### PR TITLE
Obscure moved to interface

### LINKED ISSUE #30: move input listeners into separate UI section of code
The code for #23 and #25 is really just the interface, not the core functionality of the library. We should move it into a submodule called Interface.js and require it into `BlurredLocation.Interface` to keep our code better separated and organized.

### PR DESCRIPTION
**Pull request by mridulnagpal**
closes #30

### CODE REVIEW

**Comment by jywarren** [comment]
Needs rebase, but I actually also think you should solve this by putting this:

```
setBlurred(document.getElementById("obscureLocation").checked);
```

directly into the example file, and not creating a new feature -- that's what `setBlurred()` is for, right? so can you just make this only specific to the example? Thanks!

**Comment by mridulnagpal** [comment]
Done

File: src/ui/Interface.js [hunk 1]
```diff
@@ -19,8 +19,14 @@ module.exports = function Interface (options) {
 
   panMapWhenInputsChange();
 
+  function obscureLocation() {
```
Comment (jywarren) [inline src/ui/Interface.js]: I think that makes this line unnecessary now -- looking good though! Almost there.

Comment (mridulnagpal) [inline src/ui/Interface.js]: Done, I removed it but didn't save :-P

**Pull request merged**